nixos/lib/make-options-doc/default.nix

```diff
@@ -193,12 +193,16 @@ rec {
   optionsCommonMark =
     pkgs.runCommand "options.md"
       {
+        __structuredAttrs = true;
         nativeBuildInputs = [ pkgs.nixos-render-docs ];
+        # For overriding
+        extraArgs = [ ];
```

Comment:
I don't see this overridden anywhere in this PR, is the intention to do so in a follow-up?

Reply:
`optionsCommonMark` has no in-tree usage at this time.

I will use it in https://github.com/hercules-ci/flake.parts-website, which currently relies on a rather outdated branch to provide this functionality.

Arguably this should be used in a test, but as of yet, we don't have that.

Reply:
Basic test added in fb5cff5916a3f3c28ea290cabf7f83a27a83a0d9